practice/pyramid/site.css

```diff
@@ -0,0 +1,26 @@
+div {
+  border: 1px solid black;
+}
+
+.row {
+  margin: 20px;
+  display: flex;
+  height: 100px;
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+div.inner {
+  flex: 1;
+  margin: 10px;
+}
+
+div.inner.greater{
+  flex: 4;
+}
+
+div.inner.small{
+  max-width: 90px;
+  justify: center;
```

Comment:
This isn't a valid property name. Look in the style tab of the web inspector at any warnings you get. This one says this rule isn't being used. Discover why and remove or change it.